src/Mealz/MealBundle/Resources/views/Meal/partials/meal.html.twig

```diff
@@ -13,8 +14,9 @@
     {% endif %}
 {% endif %}
 
-<div class="meal-row combined-meal"
+<div class="meal-row{{ combinedDish is same as('1') ? 'combined-meal' }} "
```

Comment:
The check is dish.isCombinedDish ? 
combinedDish is the string for the data-combined attribute.

Reply:
combinedDish will work as well, but former one is simpler. So, I changed it.